### DISCUSSION TITLE
Support for making/breaking map connections

### DISCUSSION THREAD

**Discussion by Petethegoat** [FakeFishGames/Barotrauma#10269] [Ideas and suggestions]
Similarly to how passages between zones can be blocked, let arbitrary map connections be added or destroyed (or, hidden and unhidden for use).

This opens up several possibilities for new events and mission types with major lasting effects on the game world.
For example, separatists want you to lay a massive explosive charge to collapse the passage between X & Y coalition outposts. Big payout, big reputation changes, permanently removes that connection from the map.
Or simpler things like a veteran captain in an outpost offers to share an old route to X outpost, for a small fee. This then creates, or unhides a map connection.

**Comment by MapleWheels** [comment]
Really good idea. Modders will have a lot more they can do with this.
